.github/workflows/phylogenetic.yaml

```diff
@@ -95,6 +95,7 @@ jobs:
           phylogenetic \
             deploy_all \
             --configfile build-configs/nextstrain-automation/config.yaml \
+            --set-threads align=1 tree=1 \
```

Comment:
Neat!

Reply:
Note that for the use-cases where you run (e.g.) only one serotype via the github action, using >1 cores for the tree/align should be faster. We can tweak the actions config to facilitate this, so if this is something you frequently do then I'd think the additional config complexity would be worth it.

Reply:
I'll go ahead and merge as is later today.

On a different day, I'll circle back to adding benchmarks, before I get to splitting out the config. Code development-wise, any way to speed up the 4hr build to test new PRs would be great.

Reply:
> Code development-wise, any way to speed up the 4hr build to test new PRs would be great.

We could expand the `inputs`/actions config to allow you to (i) downsample to (e.g.) 400 sequences, (ii) run only genome vs gene builds, (iii) run only a set of serotypes (I thought this was already available, but I misread) etc. You could also farm out to a big AWS box and parallalise the builds a lot.

Reply:
That makes sense, added a [tracking issue for this](https://github.com/nextstrain/dengue/issues/107). Feel free to edit the description if I missed a nuance.